source/includes/api/_areas_v2.md

```diff
@@ -257,17 +324,17 @@ The following parameters are provided to the email service and can be used in th
 * `id` : the ID of the area.
```

Comment:
A little confused about this section. What does it mean that the parameters are "provided to the email service"? Do I specify them somewhere? If I'm a user creating an area, not sure how this is relevant to me.

Reply:
These are the variables available for interpolation for someone building Sparkpost templates. This is just informative for someone interacting with the templates in Sparkpost, not useful for the "normal" API user. I've added some more information about this process above which hopefully should clarify this part.

Reply:
Seems like this is api dev documentation, not api user. if that's the case, let's keep it simple here, and maybe add a section for api devs on this

Reply:
Accepted Tiago's advice and moved the bits related to email notification parameters and all the Sparkpost information to the developer docs :)